### PR TITLE
Dark Mode Tweaks

### PR DESCRIPTION
**Pull request by grgia**
### Details

Changes made:

**Update camera icon**
image

**MenuItem highlight color**
imageimage

**Line Divider color**
image

**Match comma color**
imageimage

### Fixed Issues

$ https://github.com/Expensify/App/issues/13729
$ https://github.com/Expensify/App/issues/13735
$ https://github.com/Expensify/App/issues/13747
$ https://github.com/Expensify/App/issues/13728

### Tests

- [x] Verify that no errors appear in the JS console
- [x] Go to profile > click on avatar > ensure camera icon matches description
- [x] Go to an IOU where you owe money > click pay > ensure dropdown menu button divider line is appBG color
- [x] Go to LHN and find an unread group chat with 3+ people > ensure commas are white > click to read > ensure commas are now supporting text color
- [x] Open a menu (press + button) and ensure that the highlight color matches the description (web/desktop only)
- [x] Go to a workspace > send invoice > hover over "Send invoice" and "View all invoices" > ensure that the hover color matches the description 

### QA Steps

- [ ] Verify that no errors appear in the JS console
- [ ] Same as Tests

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

https://user-images.githubusercontent.com/38015950/209027894-ff1a2d91-c246-49eb-9cc1-1bf08e20e1ee.mov

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://user-images.githubusercontent.com/38015950/209028653-a2738e0a-3eb0-4231-b166-6fc4a2a048f0.mov

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://user-images.githubusercontent.com/38015950/209028836-8916daeb-449f-4b4d-b4e1-5993fa9817cb.mov

</details>

<details>
<summary>Desktop</summary>

https://user-images.githubusercontent.com/38015950/209028167-0e4ba3bb-8295-48fe-bfe4-1bb7b24c7663.mov

</details>

<details>
<summary>iOS</summary>

https://user-images.githubusercontent.com/38015950/209029007-a5a4e6b5-b40e-44a9-be9f-06681d59bbb9.mov

</details>

<details>
<summary>Android</summary>

</details>

https://user-images.githubusercontent.com/38015950/209028733-733ec3be-bcfb-4a3c-9296-4c17f5f9a164.mov

### CODE REVIEW

**Comment by shawnborton** [comment]
Screenshots look great, thanks for the speedy work!

**Comment by grgia** [comment]
Thoughts on this side-by-side? 
(1px vs 2px):
imageimage

**Comment by shawnborton** [comment]
I think I prefer 1px, but I don't feel too strongly.

**Comment by shawnborton** [comment]
Yeah, I think that latest version you shared looks great

**Comment by grgia** [comment]
Sweet sounds good to me! Looks like this PR is ready for final review in that case

**Review by aimane-chnaif — APPROVED** [review]
LGTM 🎉

**Review by shawnborton — APPROVED** [review]

**Comment by shawnborton** [comment]
Looks good to me too - going to go ahead and merge this as Amy is OOO.

File: src/components/MenuItem.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ const MenuItem = (props) => {
             }}
             style={({hovered, pressed}) => ([
                 styles.popoverMenuItem,
-                StyleUtils.getButtonBackgroundColorStyle(getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive)),
+                StyleUtils.getButtonBackgroundColorStyle(getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive), true),
```
Comment (rojiphil) [inline src/components/MenuItem.js:78]: Hover background colour for menu item turned out to be the same as the one for block quotes thereby causing [this issue](https://github.com/Expensify/App/issues/41901) where the quote line became invisible.

**Pull request merged**